.github/workflows/integration.yaml

```diff
@@ -12,13 +12,9 @@ concurrency:
 
 jobs:
   test:
-    name: Run e2e tests
+    name: Run integration tests
     timeout-minutes: 10
     runs-on: ubuntu-latest
-    strategy:
-      max-parallel: 2
```

Comment:
This was a bit of a bottleneck, and I think we should be able to run more tests in parallel.